packages/legend-application-query/src/application/LegendQueryApplicationConfig.ts

```diff
@@ -66,7 +66,8 @@ export interface LegendQueryApplicationConfigurationData
     TEMPORARY__useLegacyDepotServerAPIRoutes?: boolean;
   };
   engine: { url: string; queryUrl?: string };
-  studio: { url: string };
+  studio: { url: string; sdlcUrl: string };
+  extensions?: Record<PropertyKey, unknown>;
```

Comment:
```suggestion
```

This comes naturally from `LegendApplicationConfigurationData` so we can remove it here